src/bench/bench_bitcoin.cpp

```diff
@@ -25,6 +26,7 @@ static void SetupBenchArgs()
     gArgs.AddArg("-evals=<n>", strprintf("Number of measurement evaluations to perform. (default: %u)", DEFAULT_BENCH_EVALUATIONS), ArgsManager::ALLOW_ANY, OptionsCategory::OPTIONS);
     gArgs.AddArg("-filter=<regex>", strprintf("Regular expression filter to select benchmark by name (default: %s)", DEFAULT_BENCH_FILTER), ArgsManager::ALLOW_ANY, OptionsCategory::OPTIONS);
     gArgs.AddArg("-scaling=<n>", strprintf("Scaling factor for benchmark's runtime (default: %u)", DEFAULT_BENCH_SCALING), ArgsManager::ALLOW_ANY, OptionsCategory::OPTIONS);
+    gArgs.AddArg("-asymptote=n", strprintf("Asymptotic factors for benchmark's runtime, positional (default: \"\")"), ArgsManager::ALLOW_ANY, OptionsCategory::OPTIONS);
```

Comment:
Can you please add some short summary, as to what 'asymptotic factors' mean in this context? I don't think it's self-explanatory, this argument help gives me zero context on how to use it in practice :smile:

Reply:
I squash-pushed improved documentation for this parameter, which includes an example of how to use it.